core-bundle/src/Command/DeferredImagesCommand.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Contao\CoreBundle\Image\ImageFactoryInterface;
+use Contao\Image\DeferredImageInterface;
+use Contao\Image\DeferredImageStorageInterface;
+use Contao\Image\DeferredResizerInterface;
+use Contao\Image\ResizerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Terminal;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\PhpExecutableFinder;
+use Symfony\Component\Process\Process;
+
+/**
+ * Resize deferred images that have not been processed yet.
+ */
+class DeferredImagesCommand extends Command
+{
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var ImageFactoryInterface
+     */
+    private $imageFactory;
+
+    /**
+     * @var ResizerInterface
+     */
+    private $resizer;
+
+    /**
+     * @var string
+     */
+    private $targetDir;
+
+    /**
+     * @var DeferredImageStorageInterface
+     */
+    private $storage;
+
+    /**
+     * @var int
+     */
+    private $terminalWidth;
+
+    public function __construct(Filesystem $filesystem, ImageFactoryInterface $imageFactory, ResizerInterface $resizer, string $targetDir, DeferredImageStorageInterface $storage)
+    {
+        $this->filesystem = $filesystem;
+        $this->imageFactory = $imageFactory;
+        $this->resizer = $resizer;
+        $this->targetDir = $targetDir;
+        $this->storage = $storage;
+        $this->terminalWidth = (new Terminal())->getWidth();
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName('contao:deferred-images')
```

Comment:
Not sure about the command name, really. They all "do" something. `deferred-images` doesn't do anything. It should be `process-deferred-images` or anything similar?

Reply:
Maybe just `resize-images`?

Reply:
Yeah, that sounds a lot more logical to me. `deferred` is irrelevant here.

Reply:
Done in 802c3e13b253be8d305307a478156b46ba26fbfe